### PR TITLE
[release-1.3] Fixed injection and validation operations guides

### PR DESCRIPTION
**Pull request by brian-avery**
Please provide a description for what this PR is for.

This PR fixes the phrase ValidationWebhookConfiguration in the configuration operations guide to match the actual name of ValidatingWebhookConfiguration and clarifies the injection options table. 

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[x ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/en/docs/ops/setup/injection-concepts/index.md [hunk 1]
```diff
@@ -31,15 +31,11 @@ the three configuration items. The security rules above cannot be overridden.
...
 |---------------------------|------------------|---------------------------------------------------|-----------|
-| yes                       | enabled          | true                                              | yes       |
+| yes                       | enabled          | true (default)                                    | yes       |
 | yes                       | enabled          | false                                             | no        |
-| yes                       | enabled          |                                                   | yes       |
 | yes                       | disabled         | true                                              | yes       |
-| yes                       | disabled         | false                                             | no        |
-| yes                       | disabled         |                                                   | no        |
+| yes                       | disabled         | false (default)                                   | no        |
 | no                        | enabled          | true                                              | no        |
```
Comment (frankbu) [inline content/en/docs/ops/setup/injection-concepts/index.md]: Even though it doesn't change anything, I would specify (default) for the no match case too
```suggestion
 | no                        | enabled          | true  (default)                                   | no        |
```

File: content/en/docs/ops/setup/injection-concepts/index.md [hunk 2]
```diff
@@ -31,15 +31,11 @@ the three configuration items. The security rules above cannot be overridden.
...
 | no                        | enabled          | false                                             | no        |
-| no                        | enabled          |                                                   | no        |
 | no                        | disabled         | true                                              | no        |
 | no                        | disabled         | false                                             | no        |
```
Comment (frankbu) [inline content/en/docs/ops/setup/injection-concepts/index.md]:
```suggestion
 | no                        | disabled         | false  (default)                                  | no        |
```

**Comment by brian-avery** [comment]
Thanks @frankbu

File: content/en/docs/ops/setup/injection-concepts/index.md [hunk 3]
```diff
@@ -31,15 +31,11 @@ the three configuration items. The security rules above cannot be overridden.
...
-| yes                       | enabled          | true                                              | yes       |
+| yes                       | enabled          | true (default)                                    | yes       |
 | yes                       | enabled          | false                                             | no        |
-| yes                       | enabled          |                                                   | yes       |
 | yes                       | disabled         | true                                              | yes       |
-| yes                       | disabled         | false                                             | no        |
-| yes                       | disabled         |                                                   | no        |
-| no                        | enabled          | true                                              | no        |
+| yes                       | disabled         | false (default)                                   | no        |
+ | no                        | enabled          | true (default)                                    | no        |
```
Comment (frankbu) [inline content/en/docs/ops/setup/injection-concepts/index.md]:
```suggestion
| no                        | enabled          | true (default)                                    | no        |
```

File: content/en/docs/ops/setup/injection-concepts/index.md [hunk 4]
```diff
@@ -31,15 +31,11 @@ the three configuration items. The security rules above cannot be overridden.
...
 | no                        | enabled          | false                                             | no        |
-| no                        | enabled          |                                                   | no        |
 | no                        | disabled         | true                                              | no        |
-| no                        | disabled         | false                                             | no        |
-| no                        | disabled         |                                                   | no        |
\ No newline at end of file
+ | no                        | disabled         | false (default)                                   | no        |
```
Comment (frankbu) [inline content/en/docs/ops/setup/injection-concepts/index.md]:
```suggestion
| no                        | disabled         | false (default)                                   | no        |
```

**Comment by frankbu** [comment]
Sorry @brian-avery, it looks like my last suggested changes had an extra blank at the beginning of the line, which causes the table not to display correctly.

**Comment by brian-avery** [comment]
No problem. Just checked and the new version works. Thanks.

**Pull request merged**